src/viur/core/bones/boolean.py

```diff
@@ -68,7 +68,10 @@ def refresh(self, skel: 'viur.core.skeleton.SkeletonInstance', boneName: str) ->
 
             :param name: The property-name this bone has in its Skeleton (not the description!)
         """
-        if not isinstance(skel[boneName], bool):
+        if self.languages:
+            for lang in self.languages:
+                skel[boneName][lang] = utils.parse.bool(skel[boneName][lang], conf.bone_boolean_str2true)
```

Comment:
if `skel[boneName]` does not contain `lang` is not set, this will clash.

Reply:
Please clarify.

Reply:
Not every `lang` must be inside the dict `skel[boneName]` (maybe someone added a new lang, or this was just omitted before). But if you iterate over every `lang in self.languages` you get a `KeyError` on `skel[boneName][lang]`.

Reply:
Ok, so your suggestion for this is simply
```suggestion
                skel[boneName][lang] = utils.parse.bool(skel[boneName].get(lang), conf.bone_boolean_str2true)
```
and we're done?

Reply:
I think this must be rather the default value
```suggestion
                skel[boneName][lang] = utils.parse.bool(skel[boneName].get(lang, self.getDefaultValue(skel)), conf.bone_boolean_str2true)
```

---

or 
```python
if lang not in skel[name]:
    continue
```


Not sure what the ViUR specification says and how this is handled in other bones...